cookbooks/cdo-postfix/attributes/default.rb

```diff
@@ -0,0 +1,7 @@
+default['cdo-postfix'] = {
+  host: 'email-smtp.us-east-1.amazonaws.com',
```

Comment:
This endpoint is publicly listed in the Amazon Simple Email Service developer guide: [Connecting to the Amazon SES SMTP Endpoint](http://docs.aws.amazon.com/ses/latest/DeveloperGuide/smtp-connect.html). I've updated the cookbook's [README](https://github.com/code-dot-org/code-dot-org/pull/6646/files#diff-627b75ba1c94161367152c39ea38819cR3) with the line, "Installs and configures Postfix mail server using an Amazon SES account", so setting this default endpoint is appropriate.

We could generalize/make-secret our coupling to SES within this cookbook, but I didn't think this was necessary since we're not using Postfix with any non-SES accounts at this point.